Celeste.Mod.mm/Patches/Monocle/Entity.cs

```diff
@@ -13,5 +14,17 @@ class patch_Entity : Entity {
         internal void DissociateFromScene() {
             Scene = null;
         }
+
+        internal void UpdatePreceder() {
+            foreach (PreUpdateComponent puc in Components) {
+                    puc.PreUpdate();
+            }
+        }
+
+        internal void UpdateFinalizer() {
+            foreach (PostUpdateComponent puc in Components) {
```

Comment:
Cause 'System.InvalidCastException' when casting from Monocle.Component to Celeste.Mod.Entities.PostUpdateComponent